core/objects.go

```diff
@@ -250,6 +250,12 @@ type Challenge struct {
 	R     string `json:"r,omitempty"`
 	S     string `json:"s,omitempty"`
 	Nonce string `json:"nonce,omitempty"`
+
+	// IP addresses resolved from authorization identifier during SimpleHTTP validation
+	ResolvedAddrs []net.IP `json:"resolvedAddrs,omitempty"`
+
+	// URLs redirected to during SimpleHTTP validation
+	Redirects []string `json:"redirects,omitempty"`
```

Comment:
Agree we should suppress these in the output to the client until they are included in the spec (filed: https://github.com/letsencrypt/acme-spec/issues/205).